core-plugin/src/com/google/gct/idea/debugger/CloudDebuggerClient.java

```diff
@@ -49,14 +51,15 @@
   private static final Logger LOG = Logger.getInstance(CloudDebuggerClient.class);
   private static final String ROOT_URL = "https://clouddebugger.googleapis.com";
   private static final ConcurrentHashMap<String, Debugger> myDebuggerClientsFromUserEmail =
-    new ConcurrentHashMap<String, Debugger>();
+      new ConcurrentHashMap<String, Debugger>();
+  private static final String USER_AGENT = "GCP Plugin for IntelliJ";
```

Comment:
Not at all. I just had to pick something. I asked apiary/one platform folks if they had a preferred format, but they haven't responded.